### PR TITLE
Create memes.js

### PR DESCRIPTION
**Pull request by ghost**
I hand picked every meme such that they are age appropriate and there is always room go back and add memes in, While this isn't a huge feature that every server needs now a days that will either make or break it, there is no real negatives towards this plugin and its just something cool that the people could mess around with. To prevent spam, i took the entitative to have the plugin resize the memes to 250px both in height and width so that if someone were to spam it, it wouldn't be that bad. I'd appreciate it if you add this command, thank you for your time and have a good day n_n

### CODE REVIEW

File: chat-plugins/memes.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+'http://files.shandymedia.com/images/body/thesavory/weknowmemes.com_.jpg',
+'http://www.freelargeimages.com/wp-content/uploads/2014/11/Funny_Memes_7412891.jpg',
+'http://memesvault.com/wp-content/uploads/Pokemon-Meme-19.jpg',
+'http://img.memecdn.com/pokemon_o_798061.jpg',
+
+]
+exports.commands = {
+    meme : function(target, room, user){
+        if (!this.runBroadcast()) return
+    var hax = Math.floor(Math.random() * memes.length);
```
Comment (panpawn) [inline chat-plugins/memes.js:21]: why?

File: chat-plugins/memes.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/*
```
Comment (panpawn) [inline chat-plugins/memes.js:1]: `'use strict';`

File: chat-plugins/memes.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+Meme Plugin By Frete
+Made On 6/19/2016
+*/
+var memes =
```
Comment (panpawn) [inline chat-plugins/memes.js]: `const memes = [`

File: chat-plugins/memes.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
...Di9gLiCWBjgSxvGSBkjWpGcd8ZrMXXCr6XmAxQQyScgyTmZpY29mbVGEiCKyhjnOTtk9a2YpO6kOcdqZj9gGZWdb0uji0hGmc3DqbvOuTlmMaDyvKo2O4z7qXThN0RK/LVf2m3uY4GuDINcUvMm+8KDRr8oxg40++tWu9FC1TsQJh4OEX6PdPyIT7VcQXJX2nHLMDq3LyYvPkIPNtjV02rl7we7uIbmOSziYT3BuUPtmOXJy40TblecAx6vQ5xW2nOxokWwFdsRhjrHw/dmAQSrEvLsZ7NZFkL8x5go5mkIOWAUzjfr7qrPFl7eW8DyS2iaWWyjJS51kezTF9RHKGzZx7vnXo1L8Vs0mheKQZR1YMPlXbTTxT7SuHvOF4gLdYhMMOy3BnDllURkjQojwFI27/CmX+0sO78yzYxPHaqVWYCQPauZFbJTGknAI67HrmrjwMgnsbu1l80SFwgPUDSzDB9zKCPnXlDD+dDtCo3dj9q2l3kltCze0vdQ6JgArvEYtD5TzLpPUYOT0qi4J41eGQSNEHPMcuFYR6onWbyZ0nLK9wWBPZEFZ0qD1qHKGa7aYepeHuOc+5lmjCM1yrxPatLy2EZSJAUkKYZ8RZO2PMfSnvGt/crb3bSwRBJ4oYMLMXMKxk41fdgSE6idiAOm+M1nfs1iVFnuABzI1whO+M7Ej34rc8QUmFWLNlmUNnBGDnIwRih27XXuHwzxzxb4kW8jso+SVFrAsBJcNzNIQZGFGn8B9etaXxP9qIvLae2NppWRYQh5i5VomDZYhfMMqMem9X3HPA9nyywVg3qCATn12xXm/EOFohwC3frj+la40BRY+I/F3tfEYb0QFeWIBytYYtymLfi07ZzjpW/8Q+Lbm1IE1quJ1vFRUuAxU3DREFvuxjTpYe8t7q8z8M2qteQKwyDKmfrmtb4+kL8TgRvwqsWB/zEk/mB9K6uDjY8A4TcxxW8MixhLe3uIUkWQsZDOU0kpoHLwF9T1of+BXEqwmSCESWsFrFATLr1Nbyo7sDoHJ1CMDud/dWox5R8P5U9bDambTGZ0R3aSXE0NjAktw0LSu93zFPJ2XymHby5Ax0JzX37ZrLvZwt+0e0xKLvGiTl8shzysMpyx2336VpHrgrNiOMdccIupFgSW3h5kEzypcrcsFVpZNcjCHl+YYJAUmtgUFdWpAVqjRh//9k=',
```
Comment (panpawn) [inline chat-plugins/memes.js]: sir this is a local image

File: chat-plugins/memes.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+'https://pbs.twimg.com/profile_images/378800000848537568/164b936b01b7f48b32b731792ed627e9_400x400.jpeg',
+'http://files.shandymedia.com/images/body/thesavory/weknowmemes.com_.jpg',
+'http://www.freelargeimages.com/wp-content/uploads/2014/11/Funny_Memes_7412891.jpg',
+'http://memesvault.com/wp-content/uploads/Pokemon-Meme-19.jpg',
+'http://img.memecdn.com/pokemon_o_798061.jpg',
+
```
Comment (panpawn) [inline chat-plugins/memes.js]: put `];` on this line

File: chat-plugins/memes.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+]
```
Comment (panpawn) [inline chat-plugins/memes.js]: remove this

File: chat-plugins/memes.js (hunk 1 above)
Comment (panpawn) [inline chat-plugins/memes.js:21]: remove this

Comment (ghost) [inline chat-plugins/memes.js:21]: why not? i dont see any harm in adding this

File: chat-plugins/memes.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+exports.commands = {
+    meme : function(target, room, user){
+        if (!this.runBroadcast()) return
+    var hax = Math.floor(Math.random() * memes.length);
+		var lemao = memes[hax];
```
Comment (panpawn) [inline chat-plugins/memes.js:22]: remove this

File: chat-plugins/memes.js [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+    this.sendReplyBox("<img src="+lemao+" width=250 heigh=250>");
```
Comment (panpawn) [inline chat-plugins/memes.js:23]: spacing

Comment (panpawn) [inline chat-plugins/memes.js:23]: `memes.sample()`

File: chat-plugins/memes.js [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+        }
```
Comment (panpawn) [inline chat-plugins/memes.js:24]: `},`

File: chat-plugins/memes.js [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+}
```
Comment (panpawn) [inline chat-plugins/memes.js:25]: put a blank trailing line at the bottom

**Comment by panpawn** [comment]
you need to use **tabs**, not spaces

File: chat-plugins/memes.js (hunk 1 above)
Comment (panpawn) [inline chat-plugins/memes.js:21]: because it's pointless

File: chat-plugins/memes.js [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+'http://memesvault.com/wp-content/uploads/Pokemon-Meme-19.jpg',
+'http://img.memecdn.com/pokemon_o_798061.jpg',
+
+]
+exports.commands = {
+    
+    meme : function(target, room, user){
+        if (!this.runBroadcast()) return
+    var hax = Math.floor(Math.random() * memes.length);
+		var lemao = memes[hax];
```
Comment (panpawn) [inline chat-plugins/memes.js:22]: remove this

**Comment by panpawn** [comment]
do you seriously believe this is worthy of it's own plugin?

it's like a 4 or 5 lined command and an array

aka it doesn't

**Comment by ghost** [comment]
are you suggesting this being added to wisp.js?

File: chat-plugins/memes.js (hunk 8 above)
Comment (SilverTactic) [inline chat-plugins/memes.js:23]: @panpawn <code>Array.prototype.sample</code> isn't a pre-existing array method

**Comment by panpawn** [comment]
im suggesting if you want a chance of it being added, put it in `chat-plugins/wisp.js` or something

File: chat-plugins/memes.js (hunk 8 above)
Comment (panpawn) [inline chat-plugins/memes.js:23]: u sure about that @SilverTactic ?

**Comment by panpawn** [comment]
And don't make a new PR for this...

File: chat-plugins/memes.js (hunk 8 above)
Comment (SilverTactic) [inline chat-plugins/memes.js:23]: Yes

**Comment by AlainDuhHero** [comment]
how can he change the file location ? without a new PR ?

**Comment by AlainDuhHero** [comment]
u r using var
wrong eslint

**Comment by AlainDuhHero** [comment]
u better close it down
I'll PR a new meme plugin from scratch

**Comment by panpawn** [comment]
This PR looks abandoned

**Pull request closed without merging**